doc/website/FAQ.md

```diff
@@ -2,10 +2,59 @@
 
 In this document are tips and hints documented which can help for troubleshooting on RouDi.
 
-## Does iceoryx run in a docker environment
+## Does iceoryx run in a docker environment?
 
 Yes. Take a look at the [icedocker example](../../iceoryx_examples/icedocker/)
 
+## How can I find out if RouDi is running?
+
+RouDi uses a file locking machanism to ensure that only one RouDi instance is running at a time. For that RouDi
+creates and locks `/tmp/roudi.lock`. The file exists also also when RouDi is not running. Try locking this file,
+if this fails RouDi is running.
+
+On the command line do
+
+```console
+flock -n /tmp/roudi.lock echo "RouDi is not running"
+```
+
+Or for example in Python
```

Comment:
Why do we provide a python example? I would remove it, the console example is sufficient and based on it and your explanation any skilled python, perl, rust programmer can implement an equivalent in their language.